pprint/src/test/scala/ammonite/pprint/PPrintTests.scala

```diff
@@ -378,5 +378,153 @@ object PPrintTests extends TestSuite{
+        * - check(List(1,2,3,4), "List(1, 2, 3, 4)")
+        * - check(
+          List.fill(13)("asdfghjklqwertz"),
+          """List(
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz",
+            |  "asdfghjklqwertz"
+            |)
+          """.stripMargin
+        )
+      }
+
+      'short_lines_truncated{
+        implicit val cfg = Config.Defaults.PPrintConfig.copy(maxHeight = 15)
+        * - check(
+          List.fill(15)("foobarbaz"),
+          """List(
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |...""".stripMargin
+        )
+        * - check(
+        List.fill(150)("foobarbaz"),
+          """List(
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |  "foobarbaz",
+            |...""".stripMargin
+        )
+      }
+
+      'long_line_truncated{
+        implicit val cfg = Config.Defaults.PPrintConfig.copy(
+          maxWidth = 100,
+          maxHeight = 3
+        )
+        check(
+          "a" * 1000,
+          """"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa..."""
+        )
+      }
+
+      'stream{
+        implicit val cfg = Config.Defaults.PPrintConfig.copy(
+          maxHeight = 5
+        )
+        check(
+          Stream.continually("foo"),
+          """Stream(
+            |  "foo",
+            |  "foo",
+            |  "foo",
+            |  "foo",
+            |...
+          """.stripMargin
+        )
+      }
+    }
+
+    'wrapped_lines{
+      implicit val cfg = Config.Defaults.PPrintConfig.copy(
+        maxWidth = 8,
+        maxHeight = 5
+      )
+      check(
+        "1234567890\n"*10,
+        "\"\"\"\n1234567890\n1234567890\n..."
```

Comment:
Shouldn't this be
```
"\"\"\"\n12345678\n90\n12345678\n..."
```

Since the `maxWidth` is only 8, but there are 10 characters in each line of the input, and it cuts off after 5 lines